manifests/bridges.pp

```diff
@@ -1,9 +1,9 @@
 # allow forwarding traffic on bridges
-class nftables::bridges(
+class nftables::bridges (
   Enum['present','absent']
-    $ensure = 'present',
+  $ensure = 'present',
```

Comment:
I think this is less readable. Perhaps the lint plugin should understand this. Personally I don't like splitting definitions over multiple lines but I can understand why you'd prefer it over aligning. Personally I stopped aligning altogether.

Reply:
I agree both on one line and by not aligning you avoid this nasty diffs when one new longer parameter line is added.